WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -9,212 +9,75 @@
 
 #import "XCUIElement+FBIsVisible.h"
 
-#import "FBConfiguration.h"
 #import "FBElementUtils.h"
-#import "FBMathUtils.h"
-#import "FBActiveAppDetectionPoint.h"
-#import "FBSession.h"
-#import "FBXCAccessibilityElement.h"
 #import "FBXCodeCompatibility.h"
 #import "FBXCElementSnapshotWrapper+Helpers.h"
 #import "XCUIElement+FBUtilities.h"
-#import "XCUIElement+FBUID.h"
 #import "XCTestPrivateSymbols.h"
 
+#define AX_FETCH_TIMEOUT 0.3
+
+NSNumber* _Nullable fetchSnapshotVisibility(id<FBXCElementSnapshot> snapshot)
+{
+  return nil == snapshot.additionalAttributes ? nil : snapshot.additionalAttributes[FB_XCAXAIsVisibleAttribute];
```

Comment:
Btw, do you aware that where set `additionalAttributes[FB_XCAXAIsVisibleAttribute]` value?

I wondered if we needed to set the visibility/accessibility value in `additionalAttributes` in probably `fb_isAccessibilityElement`? by ourselves. XCTest framework would not set values for this `additionalAttributes`...?

So far, I haven't noticed this `additionalAttributes` had attributes so just wondered.

Reply:
good catch. Fixed